framework/_global.scss

```diff
@@ -2,17 +2,9 @@
 
 body {
   -moz-osx-font-smoothing: grayscale;
-  font-family: $font-family-base;
-  font-weight: 400;
-  line-height: 1.667;
+  -webkit-font-smoothing: antialiased;
```

Comment:
We shouldn't force antialiasing and grayscale for default font (on Windows in doesn't change anything and it consumes resources).